ADDED BOOKS AUTHORS PRICE

KapuluruBhuvaneswariVspdbct opened the pull request:
ADDED BOOKS AUTHORS PRICE

### Description
A clear and concise description of what the PR does.
- This PR does the following:
  - Adds ...
  - Fixes ...
  - Updates ...

### Related Issues
Link any related issues using the format `Fixes #issue_number`. 
This helps to automatically close related issues when the PR is merged.
- Placeholder: "Fixes #123"

### Changes
List the detailed changes made in this PR.
- Added a new feature to ...
- Refactored the ...
- Fixed a bug in ...

### Testing Instructions
Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
1. Pull this branch.
2. Run `npm install` to install dependencies.
3. Run `npm test` to execute the test suite.
4. Verify that ...

### Screenshots (if applicable)
Add any screenshots that help explain or visualize the changes.

### Additional Context
Any additional context or information that reviewers should be aware of.
- This PR is based on the following...

### Checklist
Make sure to check off all the items before submitting. Mark with [x] if done.
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I am working on this issue under GSSOC

Trisha-tech commented:
> ADDED BOOKS AUTHORS PRICE
> 
> ### Description
> A clear and concise description of what the PR does.
> 
> * This PR does the following:
>   
>   * Adds ...
>   * Fixes ...
>   * Updates ...
> 
> ### Related Issues
> Link any related issues using the format `Fixes #issue_number`. This helps to automatically close related issues when the PR is merged.
> 
> * Placeholder: "Fixes [Added workflow for closing old PRs  #123](https://github.com/Trisha-tech/OnlineBookSales/pull/123)"
> 
> ### Changes
> List the detailed changes made in this PR.
> 
> * Added a new feature to ...
> * Refactored the ...
> * Fixed a bug in ...
> 
> ### Testing Instructions
> Detailed instructions on how to test the changes. Include any setup needed and specific test cases.
> 
> 1. Pull this branch.
> 2. Run `npm install` to install dependencies.
> 3. Run `npm test` to execute the test suite.
> 4. Verify that ...
> 
> ### Screenshots (if applicable)
> Add any screenshots that help explain or visualize the changes.
> 
> ### Additional Context
> Any additional context or information that reviewers should be aware of.
> 
> * This PR is based on the following...
> 
> ### Checklist
> Make sure to check off all the items before submitting. Mark with [x] if done.
> 
> * [x]  I have performed a self-review of my code
> * [x]  I have commented my code, particularly in hard-to-understand areas
> * [x]  My changes generate no new warnings
> * [x]  I am working on this issue under GSSOC

Kindly specify the issue no you are addressing @KapuluruBhuvaneswariVspdbct.

KapuluruBhuvaneswariVspdbct commented:
Description
This PR adds functionality for managing book categories and displaying related books on dedicated pages.

This PR does the following:
Adds new book categories (Action, Thriller, Fiction, Tech, Philosophy, Manga).
Creates separate pages for each book category to display relevant books.
Integrates a feature for dynamically fetching book data from an external source (Open Library API).
Implements a responsive layout for better user experience.
Related Issues
Fixes #123 - Added workflow for closing old PRs.

Changes
Added a new feature to fetch and display books from the Open Library API based on selected categories.
Refactored the book display logic to use modular components for better readability and maintainability.
Fixed a bug in the navigation links that prevented users from accessing certain book categories.
Updated styles to enhance the UI and align with design specifications.
Testing Instructions
Pull this branch.
Run npm install to install dependencies.
Run npm start to start the development server.
Navigate to each book category page (Action, Thriller, Fiction, Tech, Philosophy, Manga).
Verify that:
Each page displays the correct list of books.
Books are displayed with the title, author, number of pages, and price.
The layout is responsive across different devices.
Screenshots (if applicable)
Include any relevant screenshots of the new pages or changes made.

Additional Context
This PR is based on the requirement to enhance the book section of the website by organizing books into categories for improved navigation and user experience. The external API integration allows for dynamic content updates and better management of book data.


Uploading bookcategories.mp4…

Trisha-tech commented:
> Description This PR adds functionality for managing book categories and displaying related books on dedicated pages.
> 
> This PR does the following: Adds new book categories (Action, Thriller, Fiction, Tech, Philosophy, Manga). Creates separate pages for each book category to display relevant books. Integrates a feature for dynamically fetching book data from an external source (Open Library API). Implements a responsive layout for better user experience. Related Issues Fixes #123 - Added workflow for closing old PRs.
> 
> Changes Added a new feature to fetch and display books from the Open Library API based on selected categories. Refactored the book display logic to use modular components for better readability and maintainability. Fixed a bug in the navigation links that prevented users from accessing certain book categories. Updated styles to enhance the UI and align with design specifications. Testing Instructions Pull this branch. Run npm install to install dependencies. Run npm start to start the development server. Navigate to each book category page (Action, Thriller, Fiction, Tech, Philosophy, Manga). Verify that: Each page displays the correct list of books. Books are displayed with the title, author, number of pages, and price. The layout is responsive across different devices. Screenshots (if applicable) Include any relevant screenshots of the new pages or changes made.
> 
> Additional Context This PR is based on the requirement to enhance the book section of the website by organizing books into categories for improved navigation and user experience. The external API integration allows for dynamic content updates and better management of book data.
> 
> Uploading bookcategories.mp4…

Issue #123 is not matching with your respective changes and video is not available. @KapuluruBhuvaneswariVspdbct

KapuluruBhuvaneswariVspdbct commented:
#351 is the  issue i solved
this is my changes video

https://github.com/user-attachments/assets/8d27231f-6c71-453a-a9f8-09ad5820470a

Trisha-tech commented:
> #351 is the issue i solved this is my changes video
> 
>  book.mp4

Kindly match the changes with the website theme which is dark blue in colour. @KapuluruBhuvaneswariVspdbct

KapuluruBhuvaneswariVspdbct commented:
please check once

KapuluruBhuvaneswariVspdbct commented:
successfully made changed colors according to the website background please have  a look at it 
<img width="959" alt="onlinequizbgchange" src="https://github.com/user-attachments/assets/6a95210d-97ed-4fa1-9124-f4cb1a5c8a11">
<img width="953" alt="image" src="https://github.com/user-attachments/assets/31122393-06cb-46bf-a774-fdf8cab6fa3d">
<img width="955" alt="image" src="https://github.com/user-attachments/assets/e434b624-7ca1-4c7e-90c1-5ce2864057fc">

KapuluruBhuvaneswariVspdbct commented:
working of quiz page 

https://github.com/user-attachments/assets/df20c503-5ad4-408d-a7bb-ae82a18e783b

Trisha-tech commented:
> successfully made changed colors according to the website background please have a look at it <img alt="onlinequizbgchange" width="959" src="https://private-user-images.githubusercontent.com/165504668/374621816-6a95210d-97ed-4fa1-9124-f4cb1a5c8a11.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIxODE2LTZhOTUyMTBkLTk3ZWQtNGZhMS05MTI0LWY0Y2IxYTVjOGExMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTc2MWY0OGM5Y2Q1YjYwZTMxNzgzZjNlNDJkZGFmNTIxYzM3YTNlMGI4YThlYmRhM2RlNjA2YzJjZTYxNzM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.appTTNnTxZojS6X7PDR3nJYIPtHd5N-LKsn4Ry1MvNU"> <img alt="image" width="953" src="https://private-user-images.githubusercontent.com/165504668/374622172-31122393-06cb-46bf-a774-fdf8cab6fa3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMTcyLTMxMTIyMzkzLTA2Y2ItNDZiZi1hNzc0LWZkZjhjYWI2ZmEzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTlkNGI3YzZlMTdhNzU5YzcwYjlkOWViY2E5ZTIyN2JjODNkZDhmZmJjYzI2MzU5MDIzZTMxMzNhODBjMWMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sCa4fZq2_LFKGDwVNZrNxrZ1P99VGXOD9CzeIymRrLM"> <img alt="image" width="955" src="https://private-user-images.githubusercontent.com/165504668/374622332-e434b624-7ca1-4c7e-90c1-5ce2864057fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMzMyLWU0MzRiNjI0LTdjYTEtNGM3ZS05MGMxLTVjZTI4NjQwNTdmYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MDI1YjE0MDA1NmJmY2RiMjVlMGZiNzY2ODcwNzMwNDIwOGU0YjgwMTNhMDNmYjUwMzU1ZGRiYzUwNTliZTMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aaf0ofJEBQlVI_wLowtR5ZO9z5liyHhwnGJxYdezOGo">

No need to add blue color in whole background. Simply put buttons and heading text in blue color. @KapuluruBhuvaneswariVspdbct

KapuluruBhuvaneswariVspdbct commented:
Mam please have a look and conside my contribution by adding large number
of books and a quiz page and categorical wise pages creation for seperate
books as level2 or level 3 give badges mam

On Wed, Oct 9, 2024, 6:06 PM Trisha Sahu ***@***.***> wrote:

> successfully made changed colors according to the website background
> please have a look at it [image: onlinequizbgchange]
> <https://private-user-images.githubusercontent.com/165504668/374621816-6a95210d-97ed-4fa1-9124-f4cb1a5c8a11.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIxODE2LTZhOTUyMTBkLTk3ZWQtNGZhMS05MTI0LWY0Y2IxYTVjOGExMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTc2MWY0OGM5Y2Q1YjYwZTMxNzgzZjNlNDJkZGFmNTIxYzM3YTNlMGI4YThlYmRhM2RlNjA2YzJjZTYxNzM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.appTTNnTxZojS6X7PDR3nJYIPtHd5N-LKsn4Ry1MvNU> [image:
> image]
> <https://private-user-images.githubusercontent.com/165504668/374622172-31122393-06cb-46bf-a774-fdf8cab6fa3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMTcyLTMxMTIyMzkzLTA2Y2ItNDZiZi1hNzc0LWZkZjhjYWI2ZmEzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTlkNGI3YzZlMTdhNzU5YzcwYjlkOWViY2E5ZTIyN2JjODNkZDhmZmJjYzI2MzU5MDIzZTMxMzNhODBjMWMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sCa4fZq2_LFKGDwVNZrNxrZ1P99VGXOD9CzeIymRrLM> [image:
> image]
> <https://private-user-images.githubusercontent.com/165504668/374622332-e434b624-7ca1-4c7e-90c1-5ce2864057fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMzMyLWU0MzRiNjI0LTdjYTEtNGM3ZS05MGMxLTVjZTI4NjQwNTdmYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MDI1YjE0MDA1NmJmY2RiMjVlMGZiNzY2ODcwNzMwNDIwOGU0YjgwMTNhMDNmYjUwMzU1ZGRiYzUwNTliZTMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aaf0ofJEBQlVI_wLowtR5ZO9z5liyHhwnGJxYdezOGo>
>
> No need to add blue color in whole background. Simply put buttons and
> heading text in blue color. @KapuluruBhuvaneswariVspdbct
> <https://github.com/KapuluruBhuvaneswariVspdbct>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Trisha-tech/OnlineBookSales/pull/355#issuecomment-2402204694>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BHOWNHGUMGCBWGJMC3M5QTDZ2UPOLAVCNFSM6AAAAABPQMZ4L6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMBSGIYDINRZGQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

Trisha-tech commented:
> Mam please have a look and conside my contribution by adding large number of books and a quiz page and categorical wise pages creation for seperate books as level2 or level 3 give badges mam
> […](#)
> On Wed, Oct 9, 2024, 6:06 PM Trisha Sahu ***@***.***> wrote: successfully made changed colors according to the website background please have a look at it [image: onlinequizbgchange] <https://private-user-images.githubusercontent.com/165504668/374621816-6a95210d-97ed-4fa1-9124-f4cb1a5c8a11.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIxODE2LTZhOTUyMTBkLTk3ZWQtNGZhMS05MTI0LWY0Y2IxYTVjOGExMS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0xZTc2MWY0OGM5Y2Q1YjYwZTMxNzgzZjNlNDJkZGFmNTIxYzM3YTNlMGI4YThlYmRhM2RlNjA2YzJjZTYxNzM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.appTTNnTxZojS6X7PDR3nJYIPtHd5N-LKsn4Ry1MvNU> [image: image] <https://private-user-images.githubusercontent.com/165504668/374622172-31122393-06cb-46bf-a774-fdf8cab6fa3d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMTcyLTMxMTIyMzkzLTA2Y2ItNDZiZi1hNzc0LWZkZjhjYWI2ZmEzZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT03MTlkNGI3YzZlMTdhNzU5YzcwYjlkOWViY2E5ZTIyN2JjODNkZDhmZmJjYzI2MzU5MDIzZTMxMzNhODBjMWMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.sCa4fZq2_LFKGDwVNZrNxrZ1P99VGXOD9CzeIymRrLM> [image: image] <https://private-user-images.githubusercontent.com/165504668/374622332-e434b624-7ca1-4c7e-90c1-5ce2864057fc.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjg0Nzc0NTYsIm5iZiI6MTcyODQ3NzE1NiwicGF0aCI6Ii8xNjU1MDQ2NjgvMzc0NjIyMzMyLWU0MzRiNjI0LTdjYTEtNGM3ZS05MGMxLTVjZTI4NjQwNTdmYy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMDA5JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTAwOVQxMjMyMzZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT05MDI1YjE0MDA1NmJmY2RiMjVlMGZiNzY2ODcwNzMwNDIwOGU0YjgwMTNhMDNmYjUwMzU1ZGRiYzUwNTliZTMwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.aaf0ofJEBQlVI_wLowtR5ZO9z5liyHhwnGJxYdezOGo> No need to add blue color in whole background. Simply put buttons and heading text in blue color. @KapuluruBhuvaneswariVspdbct <https://github.com/KapuluruBhuvaneswariVspdbct> — Reply to this email directly, view it on GitHub <[#355 (comment)](https://github.com/Trisha-tech/OnlineBookSales/pull/355#issuecomment-2402204694)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/BHOWNHGUMGCBWGJMC3M5QTDZ2UPOLAVCNFSM6AAAAABPQMZ4L6VHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDIMBSGIYDINRZGQ> . You are receiving this because you were mentioned.Message ID: ***@***.***>

Kindly pull the master branch and see your code has changed the whole directory and output of the code. Nothing is visible on the screen except your added code. Will you able to fix this or suggest any way to delete this commit and get back to previous commit @KapuluruBhuvaneswariVspdbct ?

KapuluruBhuvaneswariVspdbct commented:
kindly send ur settings.json code

harish08102004 commented:
Hey @Trisha-tech  you have tagged a wrong person.
